### PR TITLE
Added Entity Mapping trait

### PR DESCRIPTION
**Pull request by JanGalek**
resolve https://github.com/nettrine/orm/issues/33

alternative for IEntityProvider.

### CODE REVIEW

**Comment by f3l1x** [comment]
How is it working? I think it can't.

**Comment by JanGalek** [comment]
After rebase (+- 19-20h) trait will work in this way:

- find annotationDriver
- add Paths

you will use:

```php
class ForumExtension extends CompilerExtension
{
    use EntityMapping;

	public function loadConfiguration(): void
	{
		...
		$this->setEntityMappings([
                    'Forum' => __DIR__ . '/../Entity',
                ]);
	}
}
```

I have it in my project and work it ;)

File: src/DI/EntityMapping.php [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<?php declare(strict_types = 1);
+
+namespace Nettrine\ORM\DI;
+
+use Doctrine\Common\Persistence\Mapping\Driver\AnnotationDriver;
+use Nette\DI\ContainerBuilder;
+use Nette\DI\MissingServiceException;
+
+trait EntityMapping
```
Comment (f3l1x) [inline src/DI/EntityMapping.php]: Prefix with T please.

File: src/DI/EntityMapping.php [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+{
+
+	/**
+	 * @param string[] $mapping
+	 * @throws MissingServiceException
```
Comment (f3l1x) [inline src/DI/EntityMapping.php]: Drop throws

**Comment by JanGalek** [comment]
@f3l1x thanks, I will look to qa (new PR)

**Comment by JanMikes** [comment]
Hi, @f3l1x any ETA when this finds its way into stable version?

**Comment by f3l1x** [comment]
I hope soon. Stay tuned.

**Pull request merged**